utils/legacy-css/src/css/semantic.min.css

```diff
@@ -1,3 +1,6 @@
+/* This tilde syntax is used by Webpack to resolve the "inter-ui" node package in node_modules. */
+@import '~/inter-ui/inter.css';
```

Comment:
we may need to add the alias to webpack config, which comes from CRA.

Reply:
Swapped this out. We don't want to use a Webpack syntax if we can avoid it. I was kind of hoping this would be okay until we got rid of them. But it dies in Next.

The updated solution references the direct node_modules path of the CSS which should work in all environments, regardless of bundler. 👏🏽